### DISCUSSION TITLE
issues with styles after update dependencies

### DISCUSSION THREAD

**Discussion by AlaaDq** [quasarframework/quasar#13194] [General - Components / Directives / etc]
hello
after i have update the dependencies on my project i run after problem with broken layout
i was using max-width on scroll-area content but now this isn't being added to the elemnt and not working anymore
i tried to switch between object and string syntax nothing happened it only render empty style directive
![image](https://user-images.githubusercontent.com/45366622/163708908-795e63fa-c70b-4862-936b-b8971b3f7a6d.png)

i am also facing strange position of q-page-sticky elemnt after this update too
the layout of page is now broken 
maybe it is related to q-layout or q-drawer or q-page-sticky and q-scroll-area

**Comment by MilosPaunovic** [comment]
Please provide a minimal code reproduction for someone to be able to take a closer look.

**Reply by AlaaDq** [reply]
@MilosPaunovic  
i tried to spin a docker container with the new dependencies and pull a working code from main repo
and bom the layout is broken 
but if i run the same code but with old dependencies and env everything back to normal 

this is the borken setup :
```json
docker file contains 
From node:16.14.2 as build-stage
RUN npm install -g npm@6.14.8 && \
    npm install && \
    npm install -g @quasar/cli

  "dependencies": {
    "@quasar/extras": "^1.0.0",
    "@vueuse/core": "^8.2.6",
    "axios": "^0.21.1",
    "core-js": "^3.6.5",
    "quasar": "^2.0.0",
    "vue": "^3.2.33",
    "vue-i18n": "^9.0.0",
    "vue-router": "^4.0.14",
    "vuex": "^4.0.1"
  },
  "devDependencies": {
    "@babel/eslint-parser": "^7.13.14",
    "@quasar/app": "^3.0.0",
    "@quasar/quasar-app-extension-dotenv": "^1.1.0",
    "copy-webpack-plugin": "^9.0.1",
    "eslint": "^7.14.0",
    "eslint-config-prettier": "^8.1.0",
    "eslint-plugin-vue": "^7.0.0",
    "eslint-webpack-plugin": "^2.4.0"
  },
  ```
  
  
  
![image](https://user-images.githubusercontent.com/45366622/163736856-7d520ca1-7a5d-498e-a793-1d05003e6c8c.png)

### this is the broken env i wish that it will be helpful



**### what the old ENV was (what changes i did):**

```json

**update node js from V14 -> V16**

i face some errors and follow stackoverflow 
they guide  me to use this command 
npm install -g npm@6.14.8
refrence :https://stackoverflow.com/questions/65783437/npm-npm-peer-dependency-conflict



 the old dependencies was
 "dependencies": {
    "@quasar/extras": "^1.0.0",
    "@vueuse/core": "^7.5.3",
    "axios": "^0.21.1",
    "core-js": "^3.6.5",
    "quasar": "^2.0.0",
    "vue-i18n": "^9.0.0",
    "vuex": "^4.0.1"
  },
  "devDependencies": {
    "@babel/eslint-parser": "^7.13.14",
    "@quasar/app": "^3.0.0",
    "@quasar/quasar-app-extension-dotenv": "^1.1.0",
    "eslint": "^7.14.0",
    "eslint-config-prettier": "^8.1.0",
    "eslint-plugin-vue": "^7.0.0",
    "eslint-webpack-plugin": "^2.4.0",
    "copy-webpack-plugin": "^9.0.1",
    "git-hours": "^1.5.0"
  },
```

the problem could be with some update of one of this components :
q-layout or q-drawer or q-page-sticky and q-scroll-area

**Reply by MilosPaunovic** [reply]
As I said, without proper code reproduction it will be hard for anyone to provide you with any assistance. You can create a new public repository with the problem and share with us.
